lib/app.js

```diff
@@ -314,12 +314,28 @@ function _fetchCn(log, cnapi, arg, lobj, datacb, cb) {
     });
 }
 
-function _fetchVm(log, vmapi, arg, lobj, datacb, cb) {
+function _fetchVm(log, vmapi, config, arg, lobj, datacb, cb) {
     mod_assert.optionalObject(arg, 'arg');
     mod_assert.optionalFunc(datacb, 'datacb');
 
     var done = false;
-    var vmapiFilter = { limit: lobj.limit, state: 'running' };
+
+    var vmapiFilter = {
+        limit: lobj.limit
+    };
+
+    // optionally include stopped instances
+    if (config.discover_include_stopped === 'true') {
```

Comment:
The JSON key name should be camelCase (i.e., `discoverIncludeStopped`). See also the sapi manifest below.